### PR TITLE
Add more apic icons

### PR DESCRIPTION
**Pull request by kristastarr**
- Add `DashboardMotion` icon animation to API Connect icons
- Add `DashboardMotion` icon animation to API Connect demo page in `/example`
- Remove unnecessary CSS in `GlobeMotion`

### CODE REVIEW

**Comment by kristastarr** [comment]
Sean, Mike, @elycheea @glapadre 
I’d love some feedback on the animation for the Dashboard icon here:  
https://deploy-preview-229--carbon-icons-motion.netlify.app/#/apic-demo

It’s one that Mike had animated previously using Lottie, and I tried to replicate with CSS to be used on the API Connect landing page.  Here’s the Lottie version that I used for reference: https://pages.github.ibm.com/Krista-Starr/carbon-icon-animations/#/lottie-demo The ease and timing is definitely different so Mike whenever you have time to share the AE file I can get the exact specs and update.  Are there any other tweaks you guys would recommend?  

I’ll be doing a few more CSS versions of Mike’s Lotties to see if I hit any limitations, but I think this is a good one for us to look at as we compare pros and cons of both approaches. After looking at both versions, do you guys have any thoughts? 

Also it needs a code review if anyone has time 😄

**Comment by elycheea** [comment]
@kristastarr Both versions are looking good so far. In the CSS version, I feel like a delay before transitioning back could be nice though. Also looks like we have the donut rotating in the opposite direction.

The Lottie version does seem to have a smoother animation at the moment, but I think with adjustments to the timing and easing, they could be comparable. Noticing Lottie lags a bit between animations, but I’m guessing that’s more related to having the same icon several times or just the prototype and not an issue with Lottie itself, but commenting here just in case there’s a limitation there.

**Comment by glapadre** [comment]
To @elycheea's point above, the Lottie version appears to have a smoother animation for now, so with small adjustments to the timing and easing these will look amazing! Question on the dashboard icon in the Lottie demo, is the pause/delay supposed to be so long? It feels off to me like I'm waiting for the icon to transition back to its original state.

**Comment by mikeolasov** [comment]
The basic setup works great, but I think for hover "loop" the motion should be updated to match the approach we've explored in other icons. Here's a reference vid/.aep
[animated_icons_dashboard_01.aep.zip](https://github.com/carbon-design-system/icons-motion/files/11124856/animated_icons_dashboard_01.aep.zip)

https://user-images.githubusercontent.com/27870459/229200249-686a0e1e-ecf1-443f-b49c-120e71e9cd4f.mp4

**Comment by kristastarr** [comment]
This one is ready for code and motion design review! 🎉
@mikeolasov Let me know what you think! Unless you have strong objections, I would like to move forward with these animations since I think they are pretty consistent with what we've been talking about.  If you see more things that need to be refined, I think it would work best if you can go ahead and open issues for them, and they can be fixed in a future PR (unless they are super small/fast 😅 )

File: src/components/ApiConnect/GlobeMotion/GlobeMotion.js
Comment (elycheea) [inline src/components/ApiConnect/GlobeMotion/GlobeMotion.js:15]: Pretty minor, but looks like we’re missing a tab here.

**Review by elycheea — COMMENTED** [review]
I’ll leave @mikeolasov to comment on the specifics of the animations, but overall I think it’s a big improvement from the last version. (Really love connection/signal, and events feels way more cohesive.)

We should check tag alignment a little bit because I caught one that was off, but that’s not enough to block this PR imo. Just a few questions before I hit approve —

- Was grid intentionally removed from the list? I still see it listed in the index, but I saw the demo tile was taken out.
- Was the globe animation meant to show another outline on top of the flip? Looks like we meant to remove some CSS based on description but want to check if the change in animation is intended too.

File: src/components/ApiConnect/GlobeMotion/GlobeMotion.js
Comment (kristastarr) [reply src/components/ApiConnect/GlobeMotion/GlobeMotion.js]: This should be fixed now! I plan to add a linter in the future to catch this kind of stuff

**Comment by kristastarr** [comment]
@elycheea The things you mentioned for Globe should be corrected now (globe rotation animation and svg tag alignment).  I removed Grid and replaced it with Apps since our product is using that one now instead.

**Review by elycheea — APPROVED** [review]

**Pull request merged**
